Project-1/README.md

```diff
@@ -76,8 +76,10 @@ T	T	T	T	T
   ```
   Note that any definitions require that V has access to the connectives and quantifiers of first-order logic, which is not specified in the prompt. I have assumed that they are available, as otherwise there is no way to define any of the formulas in the following examples. 
   
+  Also note that none of the gendered examples are definable if we allow for non-binary genders and if we read male or female as terms referring to sex rather than gender. In the first case, this would mean that we cannot assume that man and women are binary opposites to which these labels apply. In the latter, case, male or female unary relations will not refer to any of the gendered terms mentioned below.
```

Comment:
Hi josh, for Question 1, in addition to giving truth-tables, you should also say each formula is tautology, contingent, or contradiction.

Q2: (A) A simpler CNF is: (A∨C)∧(¬B∨C) -- using distribution laws.
(b) and (c) are tautology, so their CNF is any formula of the form of "p∨not p". So you can write "B∨not B" or "A∨not A" or something else.

Q3: Today our discussion results are as follows:

The following are results of our today's meeting:

(a) B(x,y)= ∃u(P(u,x)∧P(u,y)∧¬(x=u)∧¬(y=u))∧¬F(x)∧¬(x=y).

A guy cannot be a brother or parent of himself/herself.

(b) A(x,y)= ∃u∃w(P(u,y)∧P(w,x)∧P(w,u)∧¬(x=u))∧F(x)

(c) C(x,y)= ∃u∃w∃z(P(w,z)∧P(w,u)∧P(z,x)∧P(u,y)∧¬(z=u))∧¬(x=y)

(d) O(x)= ∃u(P(u,x)∧∀y(P(u,y)→x=y))

(e) T(x)= ∃u∃y∃z(¬(y=z)∧¬(x=y)∧¬(x=z)∧P(u,x)∧P(u,y)∧P(u,z)∧¬F(y)∧¬F(z))∧∀w(P(u,w)∧¬F(w)→ w=x ∨ w=y ∨ w=z)

Q4 We cannot use variables to define a general statement in description logic. My attempt for defining (a) is:

 $B ≡ M\sqcap ∃parentOf^-.(∃parentOf\ge2)$. This is my definition. It means: x is a brother of y = x is a male & x is a child of some guy, who is a parent of at least 2 children. The key point is to introduce $parentOf^-$, which is the reverse of the parenthood, namely childhood. But I'm not sure whether I am correct. Just for reference.

Q5. When you wanna say "Jason= Instance of B", this is a specific statement in ABox, and it should be written as: "Jason: B". Similarly, "Mary Ann= Parent_of Jason and Mona" should be written as "(Mary Ann, Jason): Parent_of & (Mary Ann, Mona): Parent_of". Moreover, if you wanna give a model, you should use set-theoretic language to assign a set to each of concept, role or instance. For example, $Male^\Im$={b,a}. (You cannot write a "B" in this set because B is a concept rather than an element in the domain.)

Q6.

(a) You say "The second claims that for all things such that they are Y, there is some X that has a certain relationship with it." But you should say "The second claims that for all things such that they are **X**, there is some **Y** that has a certain relationship with it."

Q7. Sorry I don't follow your graph, maybe you can use mermaid to draw a graph. Example: 
{```mermaid
graph LR
A((a)) ---B((b))
A((a)) ---C((c))
A((a)) ---A((a))
```}

Please delete parentheses and then the graph will show itself.


Q9. (a) 3.   (Pa ∧ Qa)	1, ~v
You can not say this because this is actually to apply DeMorgan's Law (b), begging the question. 

Similarly, the third step of the proof of (b) is also begging the question. 

Also, in your proofs of (c) and (d), when you use "contradiction", for example, in the sixth step of Proof (c), I suspect DeMorgan's law is implicitly used.

Reply:
Thanks for the review Karl. I did my best to incorporate what you talked about. In particular I hope I changed all of the ones in 4 to fit the syntax you mentioned. I decided to keep the interpretations with them since I am not sure if those are supposed to go with the syntactic interpretations, and they are necessary for some of them to be interpretable. Without that, C and B at least would not be interpretable, as they require establishing a non-parent relationship to convey that they are born to related parents.

Aside from that, I incorporated some of what you suggested for 3, but I kept it mostly the same as it looks like the primary difference between what you suggested and what I had was the identity and non-identity relations, which stand out to me as redundant given the different variables in question. I don't think we need to say that X does not equal Y given the fact that X and Y are already different by virtue of being different variables. There may be a possibility that X and Y are the same things, but I think that it is redundant to specify the difference given what we have in the formulations. X is not the same as y, but the different variables already give us that information.

In response to your point on Q7, I added a brief clarification regarding the arrows (they indicate the R relation and their directionality) and also adjusted (a) there, as I spotted that I forgot to specify that X2 and Y2 will also have the relation. They should be set up to convey all the R relations with that included. I would use Mermaid but I think the existing "graph" conveys the necessary information.

For Q9, I take your point about using DeMorgan's laws there, but I am keeping my proofs there as is for two reasons (one of which I mentioned when we talked about this on Thursday). First, as I said Thursday, I think it is preferable to keep the proofs simpler in this case. If I were to change the proof with this in mind, it would only add a few subproofs from conjunctions, disjunctions, and some extra double negations and would only make the proofs longer through that when they could be proved much quicker using the negated disjunction and conjunction rules. Second, I will also note that there is nothing in the instructions that says that we can't use the laws in the proof. Given that, I prefer the simplicity and speed of the proof in question over the less potentially-question-begging proofs.

 I will also note that, while I am taking you at your word that the negated conjunction and disjunction rules are De Morgan's laws, I am not sure that I am using the rules depending on each of the relevant laws when I disprove the individual ones. For instance, you note that I appeal to (b) when I prove (a), but that is not begging the question. That is just using a different law to prove the other one. They may be intended to be used as a whole system or something to that effect, but given the lack of instruction excluding this, the different laws being used to prove each other, and the benefits of simplicity and speed over a more laborious proof I have elected to keep the proofs as is.

Reply:
Good point, Josh, I like an elegant style of proof!